src/Components/GrantsFinalizeShow.js

```diff
@@ -140,12 +182,40 @@ export default function GrantsFinalizeShow(props) {
             >
               Make a Copy
             </Link>
-            <Link
+            {/* <Link
               className="btn btn-outline-dark"
               to={`/organizations/${currentOrganizationId}/grants/${grant.id}/edit`}
             >
               Edit
-            </Link>
+            </Link> */}
+            <Button onClick={handleShowGrantCopyModal}>Copy</Button>
+            <Button onClick={handleShowGrantEditModal}>Edit</Button>
+            <Modal
+              className="modal-popup"
+              onClose={handleCloseGrantEditModal}
+              show={showGrantEditModal}
+            >
+              <Card>
+                <Card.Body>
+                  <GrantEditForm
+                    grant={grant}
+                    onSubmit={handleSubmitEditGrantForm}
+                    onCancel={handleCancelEditGrantForm}
```

Comment:
what do you think of reusing close here for cancel instead of having two separate ones?
```suggestion
                    onCancel={handleCloseGrantEditModal}
```

Reply:
I like it! And you're right, it doesn't need the same specificity.